src/common/utils/auth/getUserOrServerRedirect.ts

```diff
@@ -1,20 +1,30 @@
-import { createClient } from '@/common/services/auth/supabase/server'
+import { createClient } from '@/common/data/supabase/server'
```

Comment:
Not to keep bikeshedding on naming and placement in directories here, but I was thinking perhaps we consider external connections as "services" (e.g. if we had things like AWS RDS, Firebase, Supabase, Datadog, Sentry) and put all the client creators in there under `@/common/services/...` so according to that thinking, we should move this under `@/common/services/supabase/server`. It's also okay if you don't want to do that before merge.